kategory-core/src/main/kotlin/kategory/data/ListKW.kt

```diff
@@ -59,17 +59,17 @@ data class ListKW<out A> constructor(val list: List<A>) : ListKWKind<A>, List<A>
             return ListKW(buf)
         }
 
-        fun functor(): ListKWHKMonadInstance = ListKW.monad()
+        fun functor(): ListKWMonadInstance = ListKW.monad()
```

Comment:
Why haven't you deleted these methods too like in `Id` or `NonEmptyList`?

Reply:
I though I did, but will look over before merging